README.md

```diff
@@ -32,7 +32,7 @@ you could lint your package like this:
 
     $ dartanalyzer --options analysis_options.yaml .
     
-and see any violations of the `annotate_overrides`, `hash_and_equals`, and `prefer_is_not_empty` rules in the console.  To help you choose the rules you want to enable for your package, we have provided both a [complete list of rules][lints] and a list of the [lints that are enforced internally at Google][google-lints].
+and see any violations of the `annotate_overrides`, `hash_and_equals`, and `prefer_is_not_empty` rules in the console.  To help you choose the rules you want to enable for your package, we have provided a [complete list of rules][lints], an up-to-date list of the [lints according to the Effective Dart guide][effective-dart-lints] and a list of the [lints that are enforced internally at Google][google-lints].

```

Comment:
I wonder if this doesn't suggest that the list is complete...  What about something like:

> a growing list of [lints derived from the Effective Dart guide]

or

> an evolving list of [lints that are enforce advice in the Effective Dart guide]

@bwilkerson : you're good at word-smithing... Any ideas?

Reply:
In addition to making it clear that the rule set will be expanded as new rules are added, we probably need to be clear that not everything suggested in the guide has a rule associated with it (and hence the rule set will always be incomplete in some sense).